proxy/src/http/control.rs

```diff
@@ -170,14 +187,30 @@ mod handler {
             temp_dir.path().to_path_buf()
         };
 
-        let config = coco::config::default(
-            SecretKey::new(),
-            tmp_path.to_str().expect("path extraction failed"),
-        )?;
-        let mut peer = peer.lock().await;
+        let paths = paths::Paths::from_root(tmp_path).map_err(Error::from)?;
+
+        let pw = keystore::SecUtf8::from("radicle-upstream");
+        let mut new_keystore = keystore::Keystorage::new(&paths, pw);
+        let key = new_keystore.init_librad_key().map_err(Error::from)?;
+
+        let config = coco::config::configure(paths, key.clone());
 
         let new_peer = coco::create_peer_api(config).await?;
+
+        let mut peer = peer.lock().await;
+        let mut keystore = keystore.write().await;
+
+        let mut owner = owner.write().await;
+        let new_owner = if let Some(old_owner) = &*owner {
+            let new_owner = coco::init_user(&new_peer, key, old_owner.name())?;
+            Some(coco::verify_user(new_owner).await?)
+        } else {
+            None
```

Comment:
Shouldn't we panic for now in this branch.

Reply:
I'm inclined to leave it like this because it could be the case that we're testing the user not being set :thinking: I'm also unsure of how this will _exactly_ change with the `default_rad_self` change when I do that next.

I'm aiming for compatibility in this PR :)